src/CMakeLists.txt

```diff
@@ -1,3 +1,15 @@
+add_library(mc_state_observation SHARED conversions/kinematics.cpp
+                                        odometry/LeggedOdometryManager.cpp)
+target_link_libraries(
+  mc_state_observation PUBLIC SpaceVecAlg::SpaceVecAlg
+                              state-observation::state-observation)
+install(
+  TARGETS mc_state_observation
+  EXPORT "${TARGETS_EXPORT_NAME}"
```

Comment:
For the export to be correctly installed you need to add `set(PROJECT_USE_CMAKE_EXPORT TRUE)` in the root `CMakeLists.txt` (where other `PROJECT_` variables are set at the very start)